tests/typecheck/bad/substr.scilla

```diff
@@ -0,0 +1,5 @@
+(* substr_test.scilla *)
```

Comment:
perhaps we should remove this comment? `(* substr_test.scilla *)`

Reply:
Updated